### ISSUE TITLE
Feature request: simplify tf.train.write_graph() and tf.import_graph_def()

### ISSUE THREAD

**Issue by pannous** [tensorflow/tensorflow#7083]
As can be seen in https://github.com/tensorflow/tensorflow/issues/616 the current mechanism is too painful. (Also #4044 etc )

**Comment by drpngx** [comment]
Could you summarize what the problem is, perhaps with a snippet of code? Thanks.

**Comment by pannous** [comment]
Nevermind, the old horrible workaround of #616  has been completely streamlined and simplified in one single new function `tf.train.import_meta_graph`
